### PR TITLE
Updates docstrings and some comments in library.py

### PR DESCRIPTION
**Pull request by katecoded**
## Description

- Updates existing docstrings to more closely follow google's docstring format (for consistency). 
- Adds a docstring to the Items class.
- Fixes a few typos in comments/docstrings.
- Updates some block comments to follow the PEP 8 style guide.

All of these changes are in library.py.

## To Do

- [ ] Documentation (not applicable)
- [ ] Tests (not applicable - no actual code was changed)
- [ ] Changelog entry

Does this kind of change require a changelog entry? I am happy to add one if needed. Otherwise, please let me know what you think!

### CODE REVIEW

File: beets/library.py [hunk 1]
```diff
@@ -283,21 +287,24 @@ def key(x):
...
 class FileOperationError(Exception):
     """Indicates an error when interacting with a file on disk.
+
     Possibilities include an unsupported media type, a permissions
     error, and an unhandled Mutagen exception.
     """
 
     def __init__(self, path, reason):
-        """Create an exception describing an operation on the file at
+        """Creates an exception describing an operation on the file at
```
Comment (sampsyo) [inline beets/library.py:296]: I notice that Google guide explicitly says to use descriptive verbs, which directly contradicts PEP 257's recommendation to use an imperative mood. Weird! That is going to be a _very_ hard habit for me to break, personally… and it even makes me question whether we really do want to embrace the Google style wholeheartedly.

**Review by sampsyo — COMMENTED** [review]
This all looks really great; thank you for cleaning up so many docstrings! There is one piece that left me a little worried, though: personally, I have a somewhat averse reaction to Google's recommendation for verb tense. 😬 I'd be interested in others' opinions, but maybe we could consider sticking with Google's style *except* for this anti-PEP-257 aspect that I would find hard to adopt myself?

File: beets/library.py (hunk 1 above)
Comment (katecoded) [reply beets/library.py:296]: I would be fine with going back and changing the tense if that would be preferred! Just let me know if that is something I should do - I can go with whatever style is wanted.

Comment (sampsyo) [reply beets/library.py:296]: Sure! Why don't we go back to imperative verbs while we discuss a broader policy elsewhere?

**Comment by katecoded** [comment]
I have now altered the docstrings to use imperative-style language.

**Review by sampsyo — APPROVED** [review]
Awesome! Thank you again for this cleanup & consistency enforcement!

**Pull request merged**